frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -279,24 +271,50 @@ const TestCardForm = ({
     return null;
   };
 
-  const getAOEError = () => {
+  const whichAoeFormOption = useAOEFormOption(state.deviceId, devicesMap);
+  const aoeFormToDisplay = whichAoeFormToDisplay(whichAoeFormOption, state);
```

Comment:
I don't think we have to introduce a new variable here - we're already using `whichAoeFormOption` for all the validation and I'm not seeing how the display logic would necessitate having a form option variable distinct from the existing variable. It might be easier to manage this by updating `useAOEFormOption` to receive the additional state info and handle checking whether there are any positive test results there.

such that in `useAOEFormOption` you could have

```
export const useAOEFormOption = (testFormState: TestFormState, devicesMap: DevicesMap) {

const resultHasPositive = testFormState.testResults.some(
    (x) => x.testResult === TEST_RESULTS.POSITIVE
  );
...
// logic that determines which form option

// case where form option would return HIV
// uses same expression from the `whichAoeFormToDisplay` method

return resultHasPositive ? AOEFormOption.HIV : AOEFormOption.NONE;

}

```